test/convert.js

```diff
@@ -293,3 +295,107 @@ test('gif: non-retina', async t => {
 
   t.false(meta.hasAudio);
 });
+
+test('gif: lossy', async t => {
+  const regular = await convert('gif', {
+    inputPath: input,
+    fps: 15,
+    width: 510,
+    height: 286,
+    startTime: 10,
+    endTime: 15,
+    shouldCrop: true
+  });
+
+  settings.setMock('lossyCompression', true);
+
+  const lossy = await convert('gif', {
+    inputPath: input,
+    fps: 15,
+    width: 510,
+    height: 286,
+    startTime: 10,
+    endTime: 15,
+    shouldCrop: true
+  });
+
+  t.true(
+    fs.statSync(regular).size >
+    fs.statSync(lossy).size
+  );
```

Comment:
Not sure how else to test the lossy feature, other than comparing the file sizes

Reply:
That's the only way.